content/blog/2020-06-03-how-learnr-saved-the-day/index.markdown

```diff
@@ -0,0 +1,126 @@
+
+
+## Wake up call
+
+I happily accepted to instruct the R session, because I really enjoy teaching R, and thought this short 50 minute session would be easy to figure out and wouldn't require much work. 
+
+Well, I was certainly being naive. While there are many example teaching materials for R, most is designed for adults or young adults (college students), and at the time I couldn't find any examples for  middle- or high-schoolers. Although I came across some amazing K-12 lesson plans to teach statistics on [statisticsteacher.org](https://www.statisticsteacher.org), these weren't suitable for the short 50 minutes that was allotted for my session. (The wonderful [Teacups, Giraffes, & Statistics](https://tinystats.github.io/teacups-giraffes-and-statistics/index.html) project didn't exist back then, but today it would be another wonderful resource to teach R and statistics to a young audience.)
+
+I was also worried about finding a "fun" health dataset, but talking about diabetes, cancer, or some other complex disease did not sound fun to me at all. Knowing about the unpopularity of the session in the previous year, I was starting to feel quite desperate. I believed my session was going to mirror the [bored to death classroom scenes](https://www.youtube.com/watch?v=Yk3-Vb7CH-o) from the infamous Ferris Bueller's Day Off movie.
+
+<img src="bored_classroom.jpg" width="80%" style="display: block; margin: auto;" />
+
+
+
+Luckily, I attended [Kenia Wiedemann's talk](https://www.user2019.fr/static/pres/t258353.pdf) at useR!2019. [Kenia](https://concord.org/about/staff/kenia-wiedemann/) is involved with a research project called [CodeR4MATH](https://concord.org/our-work/research-projects/computing-with-r/), which aims to integrate R programming in high school math classes. While her curriculum also included multiple sessions throughout the semester, she talked about the amazing learnr package and all the cool things you can do with it. We also got a chance to chat about teaching R to teenagers and here some of the important points I remember from our conversation:    
+  
+1. You have to minimize any unnecessary distraction. Using RStudio Cloud or RStudio IDE in this setting might not be the best choice, as there are too many things you need to clarify before getting to the actual point. This is why learnr is perfect for this setting, you can limit what they can see and do very easily, without making it too rigid or boring.
+
+1. Most teenagers are quite self-involved, you have to find a dataset that they can relate to -- if you want to get even *some* of their attention.   
+
+1. Do not be discouraged if a session doesn't go very well. Some days, they might not even feel like listening to you, just because of the way you dress up or the class they had before you.   
```

Comment:
- "dress" rather than "dress up"
- I also disagree with this advice, but I wasn't in the conversation